scss/components/_card.scss

```diff
@@ -56,6 +56,7 @@ $card-margin: $global-margin !default;
   @if $global-flexbox {
     display: flex;
     flex-direction: column;
+    flex-grow: 1;
```

Comment:
If the card is not inside a flex parent, it will naturally grow because the default flex-basis is auto. When it is placed inside a flex-parent, it will shrink. This is inconsistent behavior and the largest use case is to have the card in some sort of grid or container. To curb the inconsistency, setting flex-basis to 100% (instead of flex-grow: 1) seems like the best solution.